Backend/app/rider-platform/rider-app/Main/spec/API/ticket.yaml

```diff
@@ -224,3 +277,12 @@ apis:
         type: API.Types.UI.TicketService.TicketBookingUpdateSeatsReq
       response:
         type: APISuccess
+
+  # # cancel service /ticket/bookings/services/cancel
+  - POST:
+      endpoint: /ticket/service/cancel
```

Comment:
What is usecase of ticket service cancel?

Reply:
it is merchant side api to cancel an service category show and cancel and refund the amount of booked tickets